eng/performance/benchmark_jobs.yml

```diff
@@ -76,20 +76,23 @@ jobs:
         steps:
         - checkout: self
           clean: true
+        - ${{ if eq(parameters.osName, 'windows') }}:
+          - script: (robocopy $(Build.SourcesDirectory) $(Build.SourcesDirectory)\notLocked /E /XD $(Build.SourcesDirectory)\notLocked $(Build.SourcesDirectory)\artifacts $(Build.SourcesDirectory)\.git) ^& IF %ERRORLEVEL% LEQ 1 exit 0
```

Comment:
the if for the exit code is requried because robocopy returns 1 if  "One of more files were copied successfully"

https://blogs.technet.microsoft.com/deploymentguys/2008/06/16/robocopy-exit-codes/